frontend/Auth/ResetPasswordEmailSent.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { useHistory } from "react-router";
+import { Button, ButtonKind } from "../Styleguide/Button";
+
+export function ResetPasswordEmailSent(props) {
+  const { email } = props.location.state;
+
+  const history = useHistory();
+
+  if (!email) {
+    history.push("/login");
```

Comment:
Calling history.push() inside of the render method is not recommended, since the render method gets called many times and you only should redirect to the new url once. Instead, render react-router's Redirect component (preferred) or call history.push inside of useEffect. See https://v5.reactrouter.com/web/api/Redirect. In this case, the best thing would be to render the redirect component if a redirect is needed, else render the rest of the jsx. So an if/else would probably be best